ixmp/tests/core/test_platform.py

```diff
@@ -22,44 +22,6 @@
     from ixmp.types import PlatformInitKwargs
 
 
-@pytest.mark.usefixtures("tmp_env")
```

Comment:
I'm realizing I don't know where I picked this up from, but the general order of contents in a Python module is:

- Globals
- Classes
- Functions

Unless there's a reason to move this class to the end of the file, let's keep it where it is. That would also make it easier to see if there are changes to particular lines.